gatsby/src/pages/faq.js

```diff
@@ -2805,10 +2806,10 @@ const Faq = ({ data }) => {
                 What is Synapse?
               </h4>
               <p>
-                Synapse is a reference &quot;
+                Synapse is a &quot;
                 <a href="#definitions">homeserver</a>&quot; implementation of
                 Matrix from the core development team at matrix.org, written in
-                Python 2/Twisted. It is intended to showcase the concept of
+                Python 3/Twisted. It is intended to showcase the concept of
```

Comment:
This sentence reads a bit awkward to me. What do you think about this?

> Synapse is a reference Matrix homeserver implementation developed by the matrix.org core team, written in Python3/Twisted.

Reply:
We want to drop the “reference” qualifier (as per [this comment](https://github.com/matrix-org/matrix.org/issues/933#issuecomment-762946130)). I will integrate the rest of your suggestion.